packages/core/src/sass/app/modules/account-signup.scss

```diff
@@ -1,11 +1,6 @@
 /* @define account-signup; weak */
 .account-signup {
     padding: 1.2rem 1.2rem 0;
-    form {
-        @include mobile {
-            max-height: 36.5rem;
-        }
-    }
     & .dc-select-native {
```

Comment:
lets add qa checklist for sanity test on password modal to make sure this change doesnt break any styles